src/test/regress/sql/with.sql

```diff
@@ -1046,3 +1046,20 @@ WITH updated AS (
 	update rank_tbl set rank = 6 where id = 5 returning rank
 )
 select count(*) from rank_tbl where rank in (select rank from updated);
+
+-- shared scan over values
+--start_ignore
+drop table if exists d;
+--end_ignore
+create table d (c1 int, c2 int) distributed by (c1);
+set gp_cte_sharing = on;
+set optimizer = off;
```

Comment:
I think that this line is not needed here, because this file is only called for the postgres planner; also, it does not need to be restored at the end.